packages/core/package.json

```diff
@@ -263,6 +263,18 @@
                     "type": "object",
                     "description": "%AWS.configuration.description.lambda.recentlyUploaded%",
                     "default": []
+                },
+                "aws.accessAnalyzer.policyChecks.referencePolicyFilePath": {
+                    "type": "string",
+                    "default": "",
+                    "description": "File path or S3 path to a control policy for Custom Policy Checks.",
```

Comment:
Q: does the user need to have access to this s3 object? or is it a preregistered url?

Reply:
User should have access to this S3 object. This object is owned by customers and use for storing their reference policies